src/Sylius/Bundle/AddressingBundle/Resources/translations/messages.de.yml

```diff
@@ -32,11 +35,12 @@ sylius:
             types:
                 country: Land
                 province: Bundesland
-                zone: Versandgebiet
+                post_code: Postleitzahl
+                zone: Zone
             scope: Umfang
             scopes:
                 all: Alle
             select: Bundesland auswählen
             select_scope: Bereich wählen
         zone_member:
-            select: Bundesland auswählen
+            select: Auswählen
```

Comment:
In main repo we only take care of English translations, rest of them are managed with [Crowdin](https://crowdin.com/project/sylius). It refers to regular messages, validation messages and flashes.